src/app/index.js

```diff
@@ -69,16 +69,22 @@ angular.module('janusHangouts', ['ngAnimate', 'ngCookies', 'ngTouch',
       return $delegate;
     });
   })
-  .run(function($http, jhConfig) {
+  .run(function($rootScope,$http, jhConfig) {
+    //global function to replace the placeholders
+    $rootScope.replacePlaceholder = function(value){
```

Comment:
My point is that you don't need it to be publicly accessible, so you don't need to pollute nor inject the `$rootScope`.